airflow/models/serialized_dag.py

```diff
@@ -57,6 +53,7 @@ class SerializedDagModel(Base):
     Because reading from database is lightweight compared to importing from files,
     it solves the webserver scalability issue.
     """
+    from airflow import DAG
```

Comment:
Out of interest does this mean the `SerialiazedDagModel.DAG` is now a valid/reachable property?

Shouldn't this also be `from airflow.models.dag import DAG` as it is in airflow core?

And I think this is still evaluated at "parse time" (i.e. not delayed until a function call) so this could be at the top of the file with the rest of the imports?

Reply:
Yep. After I split it into 3 classes I got it back at the top level.

Reply:
`SerialiazedDagModel.DAG` is still a property cos of this :)

Reply:
Moved it back to the top.